Wiki/FAQ.md

```diff
@@ -0,0 +1,652 @@
+placed individually due to a vote within the blockchain. The discussed
+project is then added or deleted from the dynamic whitelist.
+
+As far as hardware requirements, projects should be able to run on both
+64 and 32 bit OSes. Some BOINC projects are limited to architecture. We
+definitely need a set of guidelines to follow before adding a project to
+the whitelist. It is favorable to choose projects that are accessible
+to the masses.
+
+## How do I delete a project from my list of CPIDs?
+
+We used to support detaching but a vote was made to calculate magnitude
+on every project you still have valid RAC on; so GRC does not support
+detaching projects (it was supposed to prevent gaming the system). There
+is no difference if you score 0 on a projects, as all are taken into
+account equally.
+
+If you insist on not being paid for a specific project, change your team
+away from team Gridcoin for that project.
+
+# Status
+
+## How can I see my Researching Status?
+
+If you have some BOINC credits, you are in team Gridcoin and your email
+is correct in the gridcoinresearch.conf file you should see a list of
+the projects you are participating in when entering "*list cpids*" to
+the debug console of the wallet (Help -\> Debug Window -\> Console). It
+may take 24-48 hours for new accounts to be populated.
+
+## What does "diff" exactly measure? What does it tell if it's high or low?
+
+Difficulty (diff) is a variable int representing how hard the PoS
+algorithm has to make the next block to keep block minting periods at a
+stable 90 seconds rate into the future.
+
+When a lot of researchers are eligible at the same time for a block,
+diff rises to slow down minting. In contrast when no one is online
+mining, diff will drop to lure in miners to keep our blocks staking at
+once per 90 seconds.
+
+Another way to look at diff is how many distinct PoR miners are eligible
+during any one minute to stake. When diff is @20, it means 20 miners
+were on-line that minute with solutions, so diff is rising to keep
+emission steady.
+
+It's very similar to PoW diff. The big advantage we have in this version
+with PoR is we can accommodate thousands of new miners without seizing
+the chain with the RSA (Research Savings Account) and the 90 seconds
+block times.
+
+## What does stake weight mean?
+
+The personal stake weight it how many coins are eligible for staking \*
+coin age for your node.
+
+The network stake weight is the sum of coin age mined in the last 72
+blocks by the network.
+
+## How to call commands when starting Gridcoin Wallet on Windows?
+
+Create shortcut that links to the actual GRC program in Program Files
+directory and add the function at the end of the target, then properly
+name them for the function to execute:
+
+Example with -rescan option: Make a new shortcut to "C:\\Program Files
+(x86)\\GridcoinResearch\\gridcoinresearch.exe" or "C:\\Program
+Files\\GridcoinResearch\\gridcoinresearch.exe"

```

Comment:
The first directory listed is invalid, we do not support 32 bit anymore

Reply:
That's not correct. We do support 32 bit windows again.
(https://github.com/gridcoin-community/Gridcoin-Research/releases/download/4.0.6.0/gridcoin-4.0.6-win32-setup.exe)

Reply:
My bad.  Either way, the 64 bit directory should be first as that is _much_ more common

Reply:
They are most likely using the 64bit wallet if they are on a 64 bit system right?